OvieMudi/Banka
# #164979533 client transactions page

#### What does this PR do?
Creates an "all-transactions" UI page where a user (client/staff) can
 - view all transactions info of all accounts owned or managed
 - navigate to a page where the user can create a new transaction
- Creates a modal for confirming a new account creation
#### How should this be manually tested?
- Navigate to the page at https://oviemudi.github.io/Banka/UI/all-transactions
- Ensure page is responsive and meets UI requirements
#### Any background context you want to provide?
A cashier will get easy access to operations on the app from this **transactions page**
#### What are the relevant pivotal tracker stories?
[#164979533](https://www.pivotaltracker.com/story/show/164979533)
